src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length">
```

Comment:
How I can change month-length?
If you read the task, you need to implement logic to change the days count by month-length modifier :
calendar--month-length_28 - 28 days in calendar 
calendar--month-length_30 - 30 days